layouts/shortcodes/docs/languages/propagation.md

```diff
@@ -0,0 +1,24 @@
+{{ $lang := .Get 0 | default "" -}}
+
+With context propagation, [Signals](/docs/concepts/signals) can be correlated
+with each other, regardless of where they are generated. Although not limited to
+tracing, context propagation allows [traces](/docs/concepts/signals/traces) to
+build causal information about a system across services that are arbitrarily
+distributed across process and network boundaries.
+
+For the vast majority of use cases, libraries that natively support
+OpenTelemetry or [instrumentation libraries](../libraries) will automatically
+propagate trace context across services for you. Only in rare cases you will
+need to propagate context manually.
+
+{{ if $lang -}}
```

Comment:
Why have conditional content based on `$lang`? The prose in the "then" clause below seems suitable in all cases.

If we don't need the "lang" parameter to the shortcode, I'd suggest that we drop it for now, and reintroduce it later if/when needed.

Reply:
The page this is referencing is /docs/concepts/context-propagation, so that would be self-referencing